src/main/java/step1/Calculator.java

```diff
@@ -0,0 +1,30 @@
+package step1;
+
+public class Calculator {
+
+    public int calculate(String[] inputValue) {
+        int result = 0;
+        for(String number: inputValue){
+            result += validateNumber(number);
+        }
+
+        return result;
+    }
+
+    public int validateNumber(String number){
+        int changedNumber = numberCheck(number);
+
+        if (changedNumber < 0){
+            throw new RuntimeException("양수만 가능합니다.");
+        }
+        return changedNumber;
+    }
+
+    public int numberCheck(String number){
+        try{
+            return Integer.parseInt(number);
+        }catch (Exception e){
```

Comment:
중괄호는 다음과 같이 띄어주시는게 관례이오니 아래 링크를 참고해주시면 감사하겠습니다 :)

https://naver.github.io/hackday-conventions-java/#braces

```suggestion
        if (changedNumber < 0) {
            throw new RuntimeException("양수만 가능합니다.");
        }
        return changedNumber;
    }

    public int numberCheck(String number) {
        try {
            return Integer.parseInt(number);
        } catch (Exception e) {
```